pages/students/e15/e15316.html

```diff
@@ -18,17 +18,17 @@
 email_faculty: e15316@eng.pdn.ac.lk
 email_personal: imsuneth@gmail.com
 
-location: Kandy
+location: Sydney, Australia | Kandy, Sri Lanka
```

Comment:
Better to include the text inside `""`, as `|` is an operator symbol in Jekyll syntax